aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -210,23 +208,22 @@ module aptos_framework::object {
         create_object_internal(creator_address, obj_addr, false)
     }
 
-    /// Create a new object from a GUID generated by an account.
-    public fun create_object_from_account(creator: &signer): ConstructorRef {
-        let guid = account::create_guid(creator);
-        create_object_from_guid(signer::address_of(creator), guid)
+    /// Create a new object by generating a random UUID based on transaction hash.
+    /// The created object is deletable as we can guarantee the same UUID can 
+    /// never be regenerated with future txs.
+    public fun create_object(creator: &signer): ConstructorRef {
```

Comment:
I just realized passing the creator signer doesn't make sense. This should take an owner address instead, to whom the newly created owner will be transferred to

Reply:
I changed the function to take `creator_address: address` as input.